acapy_agent/wallet/did_method.py

```diff
@@ -97,6 +97,13 @@ def holder_defined_did(self) -> HolderDefinedDid:
     holder_defined_did=HolderDefinedDid.NO,
 )
 
+CHEQD = DIDMethod(
+    name="cheqd",
+    key_types=[ED25519],
+    rotation=True,
+    holder_defined_did=HolderDefinedDid.ALLOWED,
+)
+
```

Comment:
Unless the Cheqd DID Method will always be available (i.e. without a plugin), we shouldn't put this here. For an example of how to register a DID Method from a plugin, see: https://github.com/Indicio-tech/acapy-did-indy/blob/main/acapy_did_web/__init__.py#L10-L11

Reply:
@dbluhm thanks for the suggestion. Moved it to plugins!